content/docs/15.how-to-guides/azure-workload-id.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Use Azure Managed Workload on Kestra
+icon: /docs/icons/azure-aks.svg
+stage: Advanced
+topics:
+  - Kestra Concepts
+  - DevOps
+  - Integrations
+editions: ["EE"]
+---
+
+How to use Azure Workload identity to provide access to resources such as Azure Key Vault in Kestra
+
+::alert{type="info"}
+Note that this page is only relevant for the Enterprise Edition of Kestra. Should you require features such as integrations with Cloud-based secret managers, please contact us on sales@kestra.io or chat with us in our Slack community.
+::
+
+## Pre-Requisites
+
+To follow this guide you will need the following
+
+1. Kestra Enterprise Edition
+2. Account with Azure
+3. Azure CLI installed
+4. Kubernetes tools (kubectl & helm)
+5. Permissions to provision the following:
+  - AKS Cluster
```

Comment:
I think it's always good to add the link to the official products page/doc, i.e., for AKS, KeyVault, etc